storybookjs/storybook
# Add type check to PropType on OneOf

Issue: [#2652](https://github.com/storybooks/storybook/issues/2652)

## What I did
Added a type check to PropType prop on OneOf component.
## How to test 
Add a story with at component that has a oneOf prop with an evaluation in the propType declaration. 
See issue for further information.

Is this testable with jest or storyshots?
No

Does this need a new example in the kitchen sink apps?
No

Does this need an update to the documentation?
No
